.github/workflows/pytest.yml

```diff
@@ -230,9 +230,14 @@ jobs:
         run: |
           pytest -rx --durations 100 -m "combinatorial" --junitxml pytest.xml tests/training_success
 
-  test-minimal-install:
-    name: Test Minimal Install
+  test-distributed-minimal-install:
```

Comment:
Please revert this. This minimal install is needed to verify that what we put in the Getting Started works as-advertised. This is not the same thing. You can add a new test if you want to test the full dependencies.